cmd/minikube/cmd/delete.go

```diff
@@ -21,18 +21,27 @@ import (
 	"os"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 	cmdUtil "k8s.io/minikube/cmd/util"
 	"k8s.io/minikube/pkg/minikube/cluster"
+	"k8s.io/minikube/pkg/minikube/config"
 	"k8s.io/minikube/pkg/minikube/machine"
 )
 
 // deleteCmd represents the delete command
 var deleteCmd = &cobra.Command{
-	Use:   "delete",
-	Short: "Deletes a local kubernetes cluster",
-	Long: `Deletes a local kubernetes cluster. This command deletes the VM, and removes all
-associated files.`,
+	Use:   "delete MINIKUBE_PROFILE_NAME",
```

Comment:
Yep, that makes sense. Consistency among the commands :)
I have updated the PR and made delete not accept any argument.